streams/src/test/java/org/apache/kafka/streams/perf/YahooBenchmark.java

```diff
@@ -136,51 +128,48 @@ private boolean maybeSetupPhaseEvents(final String topic, final String clientId,
         String[] eventTypes = new String[]{"view", "click", "purchase"};
         Random rand = new Random();
```

Comment:
This is external to your changeset, but did we consider using a set seed for our RNG?

Hopefully, the data volume is large enough that it won't matter, but I've previously opted for this so that I know I'm benchmarking over the same data set with each run.

That is, with each run that has the same parameters. The RNG is still handy because you can (deterministically) generate any shape dataset you want.

Reply:
nit: add `final` (2x)

Initialize with `new Random(System.currentTimeMillis())`